enhancements/network/on-prem-bgp-networking.md

```diff
@@ -0,0 +1,444 @@
+---
+title: on-prem-bgp-networking
+authors:
+  - "@emilienm"
+reviewers:
+  - "@cgwalters"
+  - "@cybertron"
+  - "@fedepaol"
+  - "@jcpowermac"
+  - "@patrickdillon"
+approvers:
+  - "@cgwalters"
```

Comment:
I'll be up front: I'm not really a networking person.  I'd add @sinnykumari from the MCO side on this.

Reply:
ok thanks. I'll add @sinnykumari in the next patch.

Reply:
I'd just reiterate here that unfortunately, networking is not my area of expertise.  I don't have significant input to provide here on this.

I would say it would seem to me to make sense longer term to split off a separate repository/component for this stuff, distinct from the MCO.

Reply:
@cgwalters, if that works for you, I'll remove you from reviewers/approvers based on your last comment. However your feedback/inputs would always be welcome in this enhancement. Thanks